doc/TESTS.md

```diff
@@ -20,13 +20,20 @@ rake test
 
 These tests use [rspec](http://rspec.info/) are not executed on travis, so you need to launch the execution manually.
 
-You can use the following command:
+
+## Run your tests always on your forked repo!
+
+
+You need a fixed PR open for running some tests.
+
+Repo is the value of your forked repo
```

Comment:
I'd give first the syntax, then an example:
==============================================

Syntax:
```console
 +repo=<REPO> pr_num=<PR_NUM> rspec -fd cmdline_spec.rb
```

Where ```REPO``` is your repository in format ```project/repo````and ```PR_NUM``` is an opened and mergeable Pull Request.

For example:

```console
 cd tests/spec/
repo=MalloZup/gitarro pr_num=1 rspec -fd cmdline_spec.rb
```